apps/camunda/camunda-api/ithc.yaml

```diff
@@ -14,3 +14,9 @@ spec:
           secrets:
             - name: bpm-v14-POSTGRES-PASS
               alias: spring.datasource.password
+            - name: camunda-admin-password
+              alias: CAMUNDA_ADMIN_PASSWORD
+            - name: s2s-secret-camunda-bpm
+              alias: S2S_SECRET_CAMUNDA_BPM
+            - name: app-insights-connection-string
```

Comment:
Is this not already in the base chart ?
https://github.com/hmcts/hmcts-charts/blob/master/stable/camunda-bpm/values.yaml#L14-L15

Reply:
It is yeah but the pods are suddenly complaining about the connection string: `Caused by: java.nio.file.NoSuchFileException: /mnt/secrets/camunda/app-insights-connection-string`. Just want to check if it's some issue with how the secrets are getting pulled in and mounted - can't see why it would be but the error wasn't present prior to the previous PR.

Reply:
Running `kubectl describe secretproviderclass camunda-api-java-camunda` against ithc shows that only `bpm-v14-POSTGRES-PASS` is being passed in whereas all the secrets are being passed in on demo. You would have thought it would combine the values from the base chart with the overriding values but seems not.